community/STEERING-COMMITTEE.md

```diff
@@ -1,61 +1,123 @@
+1. Steward, control access, delegate access, and establishes processes regarding,
+   all Knative project resources and has the final say in the disposition of
+   those resources.
+1. Manage the Knative brand and decide which things can be called "Knative" and
+   how that mark can be used in relation to other efforts or vendors.
+1. Act as the final escalation point for any disputes or escalations from the
+   Technical Oversight Committee.
 
-  - Define policy for the creation and administration of community groups,
-    including [Working Groups](WORKING-GROUPS.md) and Committees.
+## Committee Meetings
 
-  - Define and evolve project governance structures and policies, including
-    project role assignment and contributor promotion.
+The Knative Steering Committee meets every two weeks or as-needed.
+Given the nature of the discussions in the committee, meetings are not generally
+open the public. Questions around governance are listed as issues in the
+[docs repo](https://github.com/knative/docs), and we invite your feedback there.
 
-  - Approve members of the Tech Oversight Committee.
+## Committee members
 
-- Management of project assets
+Seats on the Steering Committee are held by a company, not by the individual.
 
-  - Control access to, establish processes regarding, and provide a final
-    escalation path for any Knative repository.
+The current membership of the Steering committee is (listed alphabetically by name):
 
-  - Guided by the TOC for normal business.
+| &nbsp;                                                   | Member         | Company | Profile                                  |
+| -------------------------------------------------------- | -------------- | ------- | ---------------------------------------- |
+| <img width="30px" src="https://github.com/dewitt.png">   | DeWitt Clinton | Google  | [@dewitt](https://github.com/dewitt)     |
+| <img width="30px" src="https://github.com/mchmarny.png"> | Mark Chmarny   | Google  | [@mchmarny](https://github.com/mchmarny) |
+| <img width="30px" src="https://github.com/isdal.png">    | Tomas Isdal    | Google  | [@isdal](https://github.com/isdal)       |
+|   | TBD-1    | Google  |        |
+|   | TBD-2    | IBM  |        |
+|   | TBD-3    | Pivotal  |        |
+|   | TBD-4    | Red Hat  |        |
 
-  - Control and delegate access to and establish processes regarding other
-    project resources/assets not covered by the above, including web sites and
-    their domains, blogs, social-media accounts, etc.
+There are 4 additional seats to fill at this time, as indiciated by TBD.
 
-  - Manage the Knative brand to decide which things can be called “Knative” and
-    how that mark can be used in relation to other efforts or vendors.
+### Allocation of seats
 
-## Committee Mechanics
+Seats on the steering committee are allocated based upon contribution
+to the project. No final decision has been made on the exact formula.
```

Comment:
Add "by a company/entity", In _upon contribution to the project by a company/entity._
Add "will be revisit as a topic of discussion for the committee as the new members join" in _the exact formula, and will be revisit as a topic of discussion for the committee as the new members join_

Reply:
Add 

- Seat allocated to a company/entity the company will nominate a candidate to be confirmed by the committee, the committee reserves the right to not confirm the nominee, in which the company would need to assigned a new nominee.